docs/project/public-signing.md

```diff
@@ -17,6 +17,7 @@ Known issues when debugging and testing public signed assemblies on .NET Framewo
 - You will not be able to load the assembly in an AppDomain where shadow copying is turned on.
 - You will not be able to load the assembly in a partially trusted AppDomain
 - You will not be able to pre-compile ASP.NET applications
+- You will not be able to load the assembly if the `app.config` file specifies `<configuration`/`<runtime>`/`<NetFx40_LegacySecurityPolicy enabled="true" />`
```

Comment:
"may" not? Remember there's some context here that we haven't identified why some environments are fine with this.